### PR TITLE
Add Cloudant Feature and Update Scraping Functions

### PR DESCRIPTION
**Pull request by AnthonyAmanse**
This commit adds a Cloudant database for storing the reviews.
This also updates some scraping functions to have a better flow for the API

I've commented out the Watson functions in the `scrapeEveryPage` function.  
This should be in the `/reviews/:reviewId`  
Will update them in next PR

Flow would be:
program-flow

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -3,10 +3,21 @@ var cheerio = require('cheerio');
...
 var app = express();
 var review_url = "https://www.amazon.com/product-reviews/";
 var review_page = "/ref=cm_cr_arp_d_paging_btm_2?pageNumber=";
 
+var cloudantURL = process.env.CLOUDANT_URL;
+var cloudant = Cloudant(cloudantURL, function(er, cloudant, reply) {
+  if (er) {
+    throw er;
+  }
+  console.log('Connected with username: %s', reply.userCtx.name);
```
Comment (stevemar) [inline app.js:16]: minor suggestion: you can add the cloudantURL here, too

File: app.js [hunk 2]
```diff
@@ -16,41 +27,92 @@ app.use(require('body-parser').json());
...
 app.listen(port, function() {
   console.log("Oder API service is in port: " + port);
 });
 
-function scrapeEveryPage(totalPages, arrayOfReviews, productId) {
+/**
+ * Scrape every page for the reviews
+ * @param {object} options - product ID of Amazon Product found in URL
+ * options = {
+ * "totalPages": 30,
```
Comment (stevemar) [inline app.js:96]: how is the number of pages passed in?

Comment (AnthonyAmanse) [reply app.js:96]: `scrapeNumberOfPages` resolves an object
```
object = {
 "productId" = "B0123",
 "totalPages" : 30
}
```
The object is then used here

Comment (stevemar) [reply app.js:96]: ah gotcha.

**Review by stevemar — APPROVED** [review]

**Comment by horeaporutiu** [comment]
Yup, looks good. I can add the rest of the Watson functions in by the end of the week.

**Review by horeaporutiu — APPROVED** [review]

**Pull request merged**